new/LIT7279QeneAlbeki.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7279QeneAlbeki" xml:lang="en" type="work">
+        <teiHeader>
+            <fileDesc>
+                <titleStmt>
+                    <title xml:lang="gez" xml:id="t1">አልብኪ፡ ነውር፡ መርዓተ፡ ሊባኖስ፡ ዳእሙ፡ አይሁድ፡ ሰብአ፡ ሀገርኪ፡ ሴሎ።</title>
+                    <title xml:lang="gez" type="normalized" corresp="#t1">ʾAlbǝki nawr marʿāta Libānos dāʾǝmu ʾAyhud sabʾa hagarki Selo ... 
+                        (qǝne of the kʷǝllǝkǝmu-type)</title><!-- With the intervention of adding one more verse break in line 5. Otherwise there are eight
+                    lines in this qene resulting in a mawaddǝs-type.-->
+                    <title xml:lang="en" corresp="#t1">You do not have disgrace, daughter of Lebanon (!), moreover the Jews are the people of your 
```

Comment:
"Bride of Lebanon"

I think መርዓተ፡ ሊባኖስ፡ is more probably a reference to Song of Songs, so we don't have to be troubled by the geographical contradiction. I would translate:

You had no stain, Bride of Lebanon, but the Jews, the people of your city Shiloh...